### ISSUE TITLE
typing.Literal import error

### ISSUE THREAD

**Issue by arthur-s** [SmileyChris/django-countries#368]
[typing.Literal](https://docs.python.org/3/library/typing.html#typing.Literal) may be used in python 3.8 and higher, it doesn't work in python3.6. To fix it need to import Literal from typing_extension (https://stackoverflow.com/a/67193166/3110300, https://github.com/python/typing/tree/master/typing_extensions)
In my free time I can prepare a PR with fix if need.

**Comment by arthur-s** [comment]
diff in import between v7.2.1 and v7.3:
```
-from typing import Callable, Dict, Iterable, List, NamedTuple, Optional, Tuple, Union
+import re
+from typing import (
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Literal,
+    NamedTuple,
+    Optional,
+    Tuple,
+    Union,
+    overload,
+)
```
`Literal` and `overload` were added.

**Comment by SmileyChris** [comment]
Thanks for the report, fixed and released as v7.3.1

**Comment by za** [comment]
Hi @SmileyChris just for my record, is this [the commit](https://github.com/SmileyChris/django-countries/commit/ee02f2405b9ab8bc5718ad691cfb112c14caf8f7) where we fixed this issue?
